app/api/chat/route.ts

```
@@ -50,15 +49,8 @@ export function OPTIONS() {
 
-export async function POST(request: Request) {
-  const { message, conversationSlug, readPageTool, guideEnabled }: ChatRequestBody = await request.json();
-
-  const authResult = await authenticateWidget(request);
-  if (!authResult.success) {
-    return corsResponse({ error: authResult.error }, { status: 401 });
-  }
-
-  const { session, mailbox } = authResult;
+export const POST = withAuth(async ({ request }, { session, mailbox }) => {
+  const { message, conversationSlug, readPageTool, guideEnabled, isToolResult }: ChatRequestBody = await request.json();
```

ESLint

[error] 'isToolResult' is assigned a value but never used. Allowed unused vars must match /^_/u.

(@typescript-eslint/no-unused-vars)